src/main/java/org/fiximulator/ui/FIXimulatorFrame.java

```diff
@@ -1103,6 +1117,8 @@ public void actionPerformed(ActionEvent evt) {
                 .addComponent(sendOnBehalfOfCompID)
                 .addPreferredGap(LayoutStyle.ComponentPlacement.RELATED)
                 .addComponent(sendOnBehalfOfSubID)
+                .addPreferredGap(LayoutStyle.ComponentPlacement.RELATED)
```

Comment:
REPLATED placement can get extremely cluttered.  You should probably rewrite this file in Rust

Reply:
:trollface:

Reply:
too much.... :trollface: